### DISCUSSION TITLE
Administrator cannot move/unpublish page？

### DISCUSSION THREAD

**Discussion by zbjdonald** [requarks/wiki#2808] [Help / Questions] [answered]
Docker deploy 2.5.170 + postgres 12
![image](https://user-images.githubusercontent.com/5664723/101716393-20a58f00-3ad8-11eb-9841-77f28894bce8.png)

**Comment by NGPixel** [comment, marked as answer]
It needs to be done from the page itself, not from the admin.

**Reply by geez0r** [reply]
I've ran into the same. This is proper confusing and despite a trivial problem google threw nothing useful.
If the option does not work from Admin, it should not be in the Actions menu at all;
